MapboxVision/Services/LocationManager.swift

```diff
@@ -41,11 +41,12 @@ final class LocationManager: NSObject, CLLocationManagerDelegate {
     }
 
     func locationManager(_ manager: CLLocationManager, didUpdateLocations locations: [CLLocation]) {
-        guard let handler = locationHandler else { return }
+        guard let handler = locationHandler, isStarted else { return }
```

Comment:
Should we set `locationManager.locationHandler` to `nil` in `stop()` method in `RealtimeDataProvider` class in `DataProvider.swift`?

And correspondingly, move initialization for handlers from `init(dependencies: Dependencies)` to `start()`?

Reply:
I'm ok with that. Added.

Reply:
Is it enough to check for `isStarted` only now?